lib/linguist/languages.yml

```diff
@@ -5580,6 +5580,15 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+vvvv:
+  type: markup
```

Comment:
Why the markup type?

Reply:
I figured that would be the right one because vvvv patches are stored as XML. Which type would you suggest?

Reply:
XML is actually classified by Linguist as `data`, as the language is used more for general purpose data serialisation than marking up readable documents.

Since you said that these files are generated via a graphical interface, it makes sense to treat them as data too, rather than human-written source code.

Reply:
Ok, I've changed it to `data` for the next commit.

Reply:
Maybe we should change this pull request to add `.v4p` as a XML extension instead of a new language. It would achieve the same effect.

Reply:
Well, the goal is to have "vvvv" show up as a language in repositories that use it. That wouldn't work that way, or would it?

Reply:
No but `data` languages don't show up in statistics either, at least not by default.